lib/active_publisher/logging.rb

```diff
@@ -0,0 +1,27 @@
+# Taken from https://github.com/mperham/sidekiq/blob/7f882787e53d234042ff18099241403300a47585/lib/sidekiq/logging.rb
+require "time"
+require "logger"
+
+module ActivePublisher
+  module Logging
+    def self.initialize_logger(log_target = STDOUT)
+      oldlogger = defined?(@logger) ? @logger : nil
+      @logger = Logger.new(log_target)
+      @logger.level = Logger::INFO
+      oldlogger.close if oldlogger && !$TESTING # don't want to close testing's STDOUT logging
+      @logger
+    end
+
+    def self.logger
+      defined?(@logger) ? @logger : initialize_logger
+    end
+
+    def self.logger=(log)
+      @logger = (log ? log : Logger.new("/dev/null"))
```

Comment:
There is a comment at the top of the file `# Taken from https://github.com/mperham/sidekiq/blob/7f882787e53d234042ff18099241403300a47585/lib/sidekiq/logging.rb`

But I think this does bring up a good question. Do we care about allowing `logger` to be nil? If we really wanted to support this use case it seems like we could just catch the `nil` value in the setter and create a `FakeLogger` object that responds to `debug`, `error`, etc but does not actually log anything